yt/yt/core/CMakeLists.txt

```diff
@@ -128,6 +128,7 @@ target_sources(yt-yt-core PRIVATE
   ${CMAKE_SOURCE_DIR}/yt/yt/core/concurrency/invoker_queue.cpp
   ${CMAKE_SOURCE_DIR}/yt/yt/core/concurrency/lease_manager.cpp
   ${CMAKE_SOURCE_DIR}/yt/yt/core/concurrency/new_fair_share_thread_pool.cpp
+  ${CMAKE_SOURCE_DIR}/yt/yt/core/concurrency/new_new_fair_share_thread_pool.cpp
```

Comment:
Maybe you should come up with a better name?

Reply:
Not sure what can be better. It is a copy of `new_fair_share_thread_pool.cpp` with tiny changes. Feel free to propose

Maybe `new_fair_share_thread_pool_with_expected_bytes.cpp`? Or `new_fair_share_thread_pool_for_io.cpp`?